model/src/main/java/de/evoila/cf/broker/exception/ServiceInstanceNotFoundException.java

```diff
@@ -10,6 +10,10 @@
 
     private static final long serialVersionUID = -5984853893472349837L;
 
+    public String getError() {
+        return "ServiceInstanceNotFound";
```

Comment:
Different than in ServiceInstanceDoesNotExistException? Is there a specific reason for this?

Reply:
To handle cases where 404 should be returned instead of 410, we can catch ServiceInstanceDoesNotExistException and throw ServiceInstanceNotFoundException.
This way we can utilize exception handling in the base controller for this case.

Reply:
My comment was about the style of returning the error. In this case, the getter holds a hard coded value but in the other exception class, a value is stored in the object, which then is returned.